Minio.Functional.Tests/FunctionalTest.cs

```diff
@@ -3109,50 +3113,39 @@ internal async static Task PresignedGetObject_Test2(MinioClient minio)
-                    }
-                    StatObjectArgs statObjectArgs = new StatObjectArgs()
+                    PutObjectArgs putObjectArgs = new PutObjectArgs()
                                                             .WithBucket(bucketName)
-                                                            .WithObject(objectName);
-                    ObjectStat stats = await minio.StatObjectAsync(statObjectArgs);
-                    PresignedGetObjectArgs preArgs = new PresignedGetObjectArgs()
-                                                                    .WithBucket(bucketName)
-                                                                    .WithObject(objectName)
-                                                                    .WithExpiry(0);
-                    string presigned_url = await minio.PresignedGetObjectAsync(preArgs);
-                }
-                catch (InvalidExpiryRangeException)
-                {
-                    new MintLogger("PresignedGetObject_Test2", presignedGetObjectSignature, "Tests whether PresignedGetObject url retrieves object from bucket when invalid expiry is set.", TestStatus.PASS, (DateTime.Now - startTime), args:args).Log();
-                }
-                RemoveObjectArgs rmArgs = new RemoveObjectArgs()
-                                                    .WithBucket(bucketName)
-                                                    .WithObject(objectName);
+                                                            .WithObject(objectName)
+                                                            .WithStreamData(filestream)
+                                                            .WithObjectSize(filestream.Length);
 
-                await minio.RemoveObjectAsync(rmArgs);
-                await TearDown(minio, bucketName);
+                    await minio.PutObjectAsync(putObjectArgs);
+                }
+                StatObjectArgs statObjectArgs = new StatObjectArgs()
+                                                        .WithBucket(bucketName)
+                                                        .WithObject(objectName);
+                ObjectStat stats = await minio.StatObjectAsync(statObjectArgs);
+                PresignedGetObjectArgs preArgs = new PresignedGetObjectArgs()
+                                                                .WithBucket(bucketName)
+                                                                .WithObject(objectName)
+                                                                .WithExpiry(0);
+                string presigned_url = await minio.PresignedGetObjectAsync(preArgs);
+            }
+            catch (InvalidExpiryRangeException)
+            {
+                new MintLogger("PresignedGetObject_Test2", presignedGetObjectSignature, "Tests whether PresignedGetObject url retrieves object from bucket when invalid expiry is set.", TestStatus.PASS, (DateTime.Now - startTime), args:args).Log();
```

Comment:
missing throw

Reply:
`TestStatus.PASS`

Reply:
the throw should be present after the line
```
string presigned_url = await minio.PresignedGetObjectAsync(preArgs);
```
as it is not expected to succeed in this case

Reply:
Changed.